pkg/reconciler/kubequota/kubequota_controller.go

```diff
@@ -286,7 +270,7 @@ func (c *Controller) startQuotaForClusterWorkspace(ctx context.Context, clusterN
 		ReplenishmentResyncPeriod: func() time.Duration {
 			return c.fullResyncPeriod
 		},
-		DiscoveryFunc:        c.dynamicDiscoverySharedInformerFactory.DiscoveryData,
+		DiscoveryFunc:        c.dynamicDiscoverySharedInformerFactory.ServerPreferredNamespacedResources,
```

Comment:
Needs to be ServerPreferredResources because we support quota of cluster-scoped too

Reply:
Done with fd037fb2723512a41e381cc80829722cdc9d59d8.